grafana-plugin/src/style/utils.css

```diff
@@ -26,6 +26,10 @@
   height: 100%;
 }
 
+.u-display-block {
```

Comment:
what the pattern/convention are you using here? I mean prefixes `u-`

Reply:
It's `u-` from `utility` - it's in the `utils.css` file